### ISSUE TITLE
[Accessibility] Overlapping Legends when text spacing is applied as per WCAG 2.1 Text Spacing(Level AA) guidelines

### ISSUE THREAD

**Issue by puneet-ahuja** [vega/vega#3892]
Legends in the charts are overlapping when we apply css as per WCAG 2.1 Text Spacing(Level AA) guidelines.

[CodePen Link](https://codepen.io/puneetahuja/pen/OJGXNVq)

[WCAG 2.1 Text Spacing (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)

Attaching a screenshot for reference.

<img width="576" alt="Screenshot 2024-03-12 at 4 47 24 PM" src="https://github.com/vega/vega/assets/15903389/820229e8-9baf-4e0a-a1c7-2f6f999cf5af">

**Comment by domoritz** [comment]
Vega does its own layout so don't modify it afterwards with css. The reason is do that we can render to e.g. Canvas. 

This will not be fixed but it seems like you want to apply text spacing which I'm not sure we support as a property yet. You could file this as an enhancement request.

**Comment by domoritz** [comment]
I'm going to close this since CSS modifications is not something we will support anytime soon but please file an enhancement request for letter spacing. 

Btw, I don't see the issue above in Safari. It should be supported, though: https://caniuse.com/css-letter-spacing. 

<img width="535" alt="Screenshot 2024-03-12 at 11 20 12" src="https://github.com/vega/vega/assets/589034/5ff6952f-5871-42e3-84da-5ab8118c0b15">
